pkg/scheme/scheme_test.go

```diff
@@ -0,0 +1,53 @@
+package scheme_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-psap/special-resource-operator/pkg/scheme"
+	buildV1 "github.com/openshift/api/build/v1"
+	imageV1 "github.com/openshift/api/image/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	secv1 "github.com/openshift/api/security/v1"
+	monitoringV1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func TestScheme(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Scheme Suite")
+}
+
+var _ = Describe("Scheme", func() {
+	Describe("AddToScheme", func() {
```

Comment:
There is still a second level Describe here

Reply:
Not sure I missed this one. Fixed - sorry.